### PR TITLE
Splitting out homepage dynamic tests

### PR DESCRIPTION
**Pull request by sherakama**

### CODE REVIEW

**Comment by sherakama** [comment]
Is this still a thing?

**Comment by jbickar** [comment]
Yeah, it's sorely needed. There's a JIRA ticket too, BASIC-1474.

**Comment by sherakama** [comment]
Riiight. Ok. I'll see about making it a thing on this re-factor.

**Comment by jbickar** [comment]
I'm not sure why I didn't just tag these, but the work is done. Also not sure why it can't merge automatically. Will test and then re-assign.

**Comment by jbickar** [comment]
@sherakama ready for testing

File: includes/features/stanford_jumpstart_home/homepage-mayfield-dynamic.feature [hunk 1]
```diff
@@ -0,0 +1,46 @@
+Feature: Mayfield Homepage Layout
+  In order to ensure that the Jumpstart "Mayfield" dynamic page layout is configured correctly
+  As an end user
+  I want to check for the existence of content that should appear
+
+  @api
```
Comment (sherakama) [inline includes/features/stanford_jumpstart_home/homepage-mayfield-dynamic.feature]: Probably should add a @dev and @destructive to this as it does configuration change.

File: includes/features/stanford_jumpstart_home/homepage-mayfield-static.feature [hunk 2]
```diff
@@ -3,17 +3,15 @@ Feature: Mayfield Homepage Layout
   As an end user
   I want to check for the existence of content that should appear
 
-  @api @javascript
+  @api @javascript @jsv
```
Comment (sherakama) [inline includes/features/stanford_jumpstart_home/homepage-mayfield-static.feature]: Probably should add a @dev and @destructive to this as it does configuration change.

File: includes/features/stanford_jumpstart_home/homepage-mayfield-static.feature [hunk 3]
```diff
@@ -3,17 +3,15 @@ Feature: Mayfield Homepage Layout
...
     Given I am logged in as a user with the "administrator" role
     And I am on "admin/stanford-jumpstart/customize-design"
     Then I press the "edit-layouts-stanford-jumpstart-home-panama-selector" button
-    Then I press the "edit-layouts-stanford-jumpstart-home-mayfield-news-events-selector" button
-    Then I should see "Customized design options saved" in the "Console" region
     Then I press the "edit-layouts-stanford-jumpstart-home-mayfield-selector" button
     Then I should see "Customized design options saved" in the "Console" region
 
-  @api @javascript
+  @api @javascript @jsv
```
Comment (sherakama) [inline includes/features/stanford_jumpstart_home/homepage-mayfield-static.feature]: ditto for anything that gets changed and saved.

File: includes/features/stanford_jumpstart_home/homepage-mayfield-static.feature [hunk 4]
```diff
@@ -21,19 +19,19 @@ Feature: Mayfield Homepage Layout
     When I press the element with css selector ".homepage-layout > input[id*='jumpstart-home-mayfield']"
     Then I should see "Customized design options saved" in the "Console" region
 
-  @api
+  @api @jsv
```
Comment (sherakama) [inline includes/features/stanford_jumpstart_home/homepage-mayfield-static.feature]: @safe and @deploy?

File: includes/features/stanford_jumpstart_home/homepage-palm-dynamic.feature [hunk 5]
```diff
@@ -0,0 +1,43 @@
+Feature: Palm Homepage Layout
+  In order to ensure that the Jumpstart "Palm" dynamic page layout is configured correctly
+  As an end user
+  I want to check for the existence of content that should appear
+
+@api
```
Comment (sherakama) [inline includes/features/stanford_jumpstart_home/homepage-palm-dynamic.feature]: indent.

Comment (sherakama) [inline includes/features/stanford_jumpstart_home/homepage-palm-dynamic.feature]: Probably should add a @dev and @destructive to this as it does configuration change.

File: includes/features/stanford_jumpstart_home/homepage-palm-static.feature [hunk 6]
```diff
@@ -3,17 +3,15 @@ Feature: Palm Homepage Layout
   As an end user
   I want to check for the existence of content that should appear
 
-  @api @javascript
+  @api @javascript @jsv
```
Comment (sherakama) [inline includes/features/stanford_jumpstart_home/homepage-palm-static.feature]: Probably should add a @dev and @destructive to this as it does configuration change.

**Comment by jbickar** [comment]
Grabbing this back in order to add the recommended tags.

File: includes/features/stanford_jumpstart_home/homepage-palm-dynamic.feature (hunk 5 above)
Comment (jbickar) [inline includes/features/stanford_jumpstart_home/homepage-palm-dynamic.feature]: I'll indent where I wanna indent.

**Comment by jbickar** [comment]
Good news, everyone!

I ran the following sets of Scenarios (with Behat 1) and they all passed:
* All `stanford_jumpstart_home` Features on a JSA site
* All `@jsa` Scenarios on a JSA site
* All `@jsv` Scenarios on a JSV site

W00t.

**Comment by sherakama** [comment]
Shea McKinney approves of this.

**Comment by sherakama** [comment]
Nice work @jbickar, this is great stuff.

**Pull request merged**